LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright 2018-2019 Open-MMLab. All rights reserved.
+Copyright (c) OpenMMLab. All rights reserved.
```

Comment:
We do not need to add this claim here.

Reply:
MMCV modified LICENSE as this style.
https://github.com/open-mmlab/mmcv/blob/7540cf73ac7e5d1e14d0ffbd9b6759e83929ecfc/LICENSE#L1